src/freertos_drivers/pic32mx/Pic32mxCan.cxx

```diff
@@ -62,8 +62,7 @@ class Pic32mxCan : public Node
         , hw_(module)
         , overrunCount_(0)
     {
-        messageFifoArea_ = malloc(
-            (config_can_tx_buffer_size() + config_can_rx_buffer_size()) * 16);
+        messageFifoArea_ = malloc(2 * 16 * CAN_TX_RX_MESSAGE_SIZE_BYTES);
```

Comment:
What is the value of config_can_rx_buffer_size()? I couldn't find it just yet. We have one (1) receive FIFO and one (1) transmit FIFO, hence the value 2. Each FIFO can have up to 32 message buffers, I have assigned a fixed 16 constant here, which is the size of the receive FIFO. Each message is 16 bytes and is given by the constant CAN_TX_RX_MESSAGE_SIZE_BYTES. The transmit FIFO only uses 1 message buffer at the moment. Do you want/need to override the number of message buffers for the receive FIFO?